arch/envoy.template.yaml

```diff
@@ -51,11 +51,11 @@ static_resources:
                         envoy_grpc:
                           cluster_name: opentelemetry_collector
                         timeout: 0.250s
-                      service_name: archgw(inbound)
+                      service_name: plano(inbound)
                   random_sampling:
                     value: {{ arch_tracing.random_sampling }}
                 {% endif %}
-                stat_prefix: ingress_traffic
+                stat_prefix: plano(inbound)
```

Comment:
do we want to name it sth that conveys that it is actually about prompt gateway like sth maybe plano(inbound, prompt)